app/parse.py

```diff
@@ -8,8 +15,46 @@ class Quote:
     tags: list[str]
 
 
+def get_single_quotes(quote_soup: Tag) -> Quote:
```

Comment:
The function get_single_quotes might be better named parse_single_quote for clarity, aligning with its purpose.